README_CHUNK_FORMAT.rst

```diff
@@ -103,40 +92,108 @@ All entries are little endian.
 :cbytes:
     (``uint32``) Compressed size of the buffer (including this header).
 
+:filter_codes:
+    (``uint8x6``) Filter code.
+
+    :``0``:
+        No shuffle (for compatibility with Blosc1).
+    :``0``:
+        No filter.
+    :``1``:
+        Byte-wise shuffle.
+    :``2``:
+        Bit-wise shuffle.
+    :``3``:
+        Delta filter.
+    :``4``:
+        Truncate precision filter.
+
+    The filter pipeline has 6 reserved slots for the filters. They are applied sequentially to the chunk according
+    to their index in increasing order. The type of filter applied is specified by the `filter_code`. Each
+    `filter_code` has an associated field in `filter_meta` that can contain metadata about the filter.
+
+:filter_meta:
+
+    (``uint8x6``) Filter metadata.
+
+    Possible metadata associated with a filter code.
+
 :blosc2_flags:
     (``bitfield``) The flags for a Blosc2 buffer.
 
     :bit 0 (``0x01``):
         Whether the codec uses dictionaries or not.
     :bit 1 (``0x02``):
-        Whether the header is extended with +32 bytes coming right after this byte. 
+        Whether the header is extended with +32 bytes coming right after this byte.
     :bit 2 (``0x04``):
-        Whether the codec is stored in a byte previous to this compressed buffer or it is in the global `flags` for chunk. 
+        Whether the codec is stored in a byte previous to this compressed buffer
+        or it is in the global `flags` for chunk.
+
+
+Blocks
+------
+
+The blocks section is composed of a list of offsets to the start of each block, an optional dictionary to aid in
+compression, and finally a list of compressed data streams::
 
+    +=========+======+========+
+    | bstarts | dict | cdatas |
+    +=========+======+========+
 
-The blocks / splits section
----------------------------
+Each block is equal-sized as specified by the `blocksize` header field. The size of the last block that can be shorter
+or equal to the rest.
 
-After the header, there come the blocks / splits section.  Blocks are equal-sized parts of the chunk, except for the last block that can be shorter or equal than the rest.
+**Block starts**
 
-At the beginning of the blocks section, there come a list of `int32_t bstarts` to indicate where the different encoded blocks starts (counting from the end of this `bstarts` section)::
+The *block starts* section contains a list of offsets `int32_t bstarts` that indicate where each block starts in the
```

Comment:
For consistency with the rest of the document, I think it is better int32_t -> int32